### DISCUSSION TITLE
Display neu starten

### DISCUSSION THREAD

**Discussion by Hotzi57** [beabel/radiowecker#74] [Fragen & Antworten / Q&A] [answered]
Erstmal großes Dankeschön für 4.0, das ist ja ganz groß!
Ich hab Probleme mit dem Display, es " verschluckt" sich ab und zu, aber schon bei den früheren Versionen.
Dann fehlen Zahlen der Uhrzeit. Mache ich irgend eine Aktion, stimmt alles wieder, aber jetzt fehlt dann noch das Wetter. Das kommt erst nach Neustart (Strom aus und an) Siehe Bilder. Könnte man irgendwie ein Reset integrieren, wi der Bildschirm nach x Minuten neu gestartet wird ( immer und regelmäßig)?

![20241117_154541](https://github.com/user-attachments/assets/9d693102-264a-4f25-a0d9-281591e10b4b)
![20241117_154605](https://github.com/user-attachments/assets/afce5543-95c6-4683-89d5-d44c1aaf5467)
![20241117_154629](https://github.com/user-attachments/assets/e3dccf92-07ca-4518-b701-38f3677628ed)

**Comment by beabel** [comment]
Das Wetter wird nur aller 1 min neu geholt, bis dahin ist es leer . Und falls es keine Daten bekommt auch leer.

**Comment by Hotzi57** [comment]
Nach einer Minute wird dann das Display automatisch aktualisiert? Das klappt aber eben nicht. Siehe 1. Bild. Die Stunde >1 ist aus, die Minuten zählen weiter. Das ist das Problem.

**Reply by beabel** [reply]
Ich werde mir mal den reconnect anschauen, denn bei dir scheint es ja einen Verbindungsabbruch gegeben zu haben.

**Comment by Hotzi57** [comment]
Stimmt, ich habe immer mal Internet bruch, leider. Da fängt sich das Gerät
nicht immer wieder.

**Reply by beabel** [reply, marked as answer]
@Hotzi57 
Ich habe mal in dem [develop_401 Fork](https://github.com/beabel/radiowecker/tree/develop_401) eine Änderung eingebaut, das nach erneuter Verbindung das Display komplett neu gezeichnet wird.
[Hier die gemachten Änderungen](https://github.com/beabel/radiowecker/compare/main...develop_401)
Bitte mal testen und berichten.

**Reply by Hotzi57** [reply]
Vielen Dank, ich teste das mal. Ich hatte schon den Neustart nach 5 Minuten Trennung auf 10 sek. geändert. 
Habe es überspielt, ich gebe Bescheid, wenn ich etwas bemerke. Im Moment läufts tadellos.

**Reply by beabel** [reply]
@Hotzi57 

Und. wie sieht der Status aus? Problem behoben?
Du solltest ja in den letzten Tagen mal wieder **_Internet-Bruch_** gehabt haben .

**Reply by Hotzi57** [reply]
Sorry, hab's nicht vergessen. Es läuft im Moment, offenbar hast du es behoben. 
Danke nach Limbach!👍🤓
